packages/dds/ink/src/inkCanvas.ts

```diff
@@ -188,23 +188,23 @@ export class InkCanvas {
 		}
 	}
 
-	private handlePointerMove(evt: PointerEvent) {
+	private handlePointerMove(evt: PointerEvent): void {
 		if (this.localActiveStrokeMap.has(evt.pointerId)) {
-			const evts = (evt as any)?.getCoalescedEvents() ?? ([evt] as PointerEvent[]);
+			const evts = evt.getCoalescedEvents();
```

Comment:
Hmm without an app that runs the ink code in several environments, some where `getCoalescedEvents()` exists and some where it doesn't, it's very hard to say if this is correct at runtime, but I'm actually leaning towards reverting this bit. I imagine the motivation for the previous code is the fact that the method might not exist at runtime, and it handles that case, but the new version of the code doesn't have such handling.

I don't know that we have many use cases for this DDS, so I think it might be better to be cautious with the changes, restore the old line, and just add a disable to the lint rule here.

Reply:
OKOK